openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -160,11 +174,8 @@ const Carousel = {
                     $.ajax({
                         url: url,
                         type: 'GET',
-                        success: function(subject_results) {
-                            var works = subject_results.works;
-                            if (!works) {
-                                works = subject_results.docs;
-                            }
+                        success: function(results) {
+                            var works = results.works || results.docs;
```

Comment:
```suggestion
                            const works = results.works || results.docs;
```
We should move away from using `var` to declare variables.  `const` is good in this case as `works` will not be reassigned.